### PR TITLE
Add Sitemap Links for Dark Mode

### PR DESCRIPTION
**Pull request by kinson**
# Overview 

Related to #109 .

Added another sitemap file for dark mode so that Rocket Validator can validate accessibility of dark mode as well.

I am open to suggestions on this, but to handle the `ui` query parameter I added a function to `LayoutView` which I use inside the `root.html.eex` file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,7 +43,8 @@ erl_crash.dump
 .LSOverride
 
 # Icon must end with two \r
-Icon
```
Comment (kinson) [inline .gitignore:45]: This looks like a line ending issue, @doomspork any idea how to undo this change specifically? I didn't update this particular line in my editor.

File: lib/school_house_web/templates/layout/root.html.leex [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="<%= current_locale() %>">
+<html lang="<%= current_locale() %>" class=" <%= render_dark_mode?(@conn) %>">
```
Comment (jaimeiniesta) [inline lib/school_house_web/templates/layout/root.html.leex:2]: Unneeded xtra space inside `class=" "`

**Review by jaimeiniesta — APPROVED** [review]
This works great, @kinson 

I didn't remember that the XML sitemap is not dynamic, but generated statically. Makes total sense, so it's a great solution to have a separate file just for the dark mode links.

I also thought that the dark mode setting would be set in a plug but the LayoutView solution works fine as well. 👍

File: .gitignore (hunk 1 above)
Comment (doomspork) [reply .gitignore:45]: My best suggestion is going to be a git reset on this file. Ping me in Slack if you want to walk through it

File: lib/school_house_web/templates/layout/root.html.leex (hunk 2 above)
Comment (doomspork) [reply lib/school_house_web/templates/layout/root.html.leex:2]:
```suggestion
<html lang="<%= current_locale() %>" class="<%= render_dark_mode?(@conn) %>">
```
